pkg/sip/participant.go

```diff
@@ -14,7 +14,22 @@
 
 package sip
 
-import "github.com/livekit/protocol/livekit"
+import (
+	"time"
+
+	"github.com/livekit/protocol/livekit"
+)
+
+const (
+	// defaultMaxCallDuration sets a default max call duration. Value is according to Twilio's default.
+	defaultMaxCallDuration = 4 * time.Hour
```

Comment:
while I think this is fine, I feel a bit weird about enforcing a default timeout on our side, when the trunking provider is already doing their own.

it seems that it could potentially confuse users where the 4h timeout is coming from. wdyt about either:
* increasing our default to something much greater: i.e. 24h (and ensure it's documented in docs)
* do not default to timing out

Reply:
> while I think this is fine, I feel a bit weird about enforcing a default timeout on our side, when the trunking provider is already doing their own.

This is mostly to protect us from wasting resources on misconfigured Cloud projects. While Twilio has the limit, other providers or self-hosted SIP instances may not have this limit.

I propose to raise it to 6h or 8h so that it's still not that long but is still distinguishable from Twilio's timeout. Note that it's not a hard limit - user can set it to a higher value, if needed.

Reply:
OK, I'll then cap the timeout at 24h max :+1: